NuKeeper/Engine/GithubEngine.cs

```diff
@@ -47,5 +54,19 @@ public class GithubEngine
                 await _repositoryEngine.Run(repository, gitCreds, userIdentity);
             }
         }
+
+        private Identity GetUserIdentity(Account githubUser)
+        {
+            try
```

Comment:
I would return null right away if `string.IsNullOrWhiteSpace(githubUser?.Name)` or the Email, and avoid throwing and catchingthe exception in that case.